src/core/download.ts

```diff
@@ -14,17 +14,20 @@ export interface DownloadError extends Error {
 }
 
 export async function downloadWithProgress(
-  url: string,
+  url: string | Response,
   msg: string,
   toFile: string,
 ) {
   // Fetch the data
-  const response = await fetch(
-    url,
-    {
-      redirect: "follow",
-    },
-  );
+  console.log(url);
```

Comment:
Stray `console.log()`

Reply:
I dropped all the other ones 🤦 fixed. sorry.